Octokit.Reactive/Clients/ObservableTeamsClient.cs

```diff
@@ -47,8 +47,22 @@ public IObservable<Team> Get(int id)
         /// <returns>A list of the orgs's teams <see cref="Team"/>s.</returns>
         public IObservable<Team> GetAll(string org)
         {
+            return GetAll(org, ApiOptions.None);
```

Comment:
I also thought of adding it, but since it will be checked in the internal `GetAll`, I skipped it. Should I add here too?